hw/xfree86/drivers/video/modesetting/driver.c

```diff
@@ -495,11 +493,15 @@ Probe(DriverPtr drv, int flags)
     }
 
     for (i = 0; i < numDevSections; i++) {
-        int entity_num;
-        dev = xf86FindOptionValue(devSections[i]->options, "kmsdev");
-        if (probe_hw(dev, NULL)) {
+        int entity_num = -1;
```

Comment:
looks like it's just moving field declaration and assignment together ... perhaps should be done by separate PR.